### PR TITLE
questdb: Add version 6.0.9

### PR DESCRIPTION
**Pull request by niheaven**

### CODE REVIEW

**Review by rashil2000 — APPROVED** [review]

File: bucket/questdb.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+    "version": "6.0.9",
+    "description": "An open source SQL database designed to process time series data, faster.",
+    "homepage": "https://questdb.io",
+    "license": "Apache-2.0",
+    "suggest": {
+        "JDK": [
+            "java/oraclejdk",
```
Comment (pratikpc) [inline bucket/questdb.json:8]: Not around any longer

Comment (pratikpc) [reply bucket/questdb.json:8]: I guess this line could be removed

File: bucket/questdb.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    "persist": [
+        "conf",
+        "db",
+        "log"
+    ],
+    "checkver": {
+        "github": "https://github.com/questdb/questdb"
+    },
+    "autoupdate": {
+        "url": "https://github.com/questdb/questdb/releases/download/$version/questdb-$version-no-jre-bin.tar.gz",
```
Comment (pratikpc) [inline bucket/questdb.json:29]: This should be in 64 bit block BTW

**Review by pratikpc — CHANGES_REQUESTED** [review]

File: bucket/questdb.json (hunk 2 above)
Comment (niheaven) [reply bucket/questdb.json:29]: This can update all architecture, but yes, move it under `arch` is okay.

Comment (pratikpc) [reply bucket/questdb.json]: If it has 32 bit support then it should be outside though.

In architecture block, you had used 64 bit, so thought it was 64 bit.

**Review by pratikpc — APPROVED** [review]

**Pull request merged**